third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2901,6 +2921,17 @@ func expandMaintenancePolicy(d *schema.ResourceData, meta interface{}) *containe
 	}
 	maintenancePolicy := l[0].(map[string]interface{})
 
+	if maintenanceExclusions, ok := maintenancePolicy["maintenance_exclusion_window"]; ok && len(maintenanceExclusions.(*schema.Set).List()) > 0 {
+		for k, me := range maintenanceExclusions.(*schema.Set).List() {
+			mew := me.(map[string]interface{})
+			blackout := fmt.Sprintf("blackout-%d", k+1)
+			exclusions[blackout] = containerBeta.TimeWindow{
```

Comment:
I feel like we should allow users to input the `name` of an exclusion window, as described here in the API guide: https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions#example-maintenance-exclusions

Can we add an `exclusion_name` or similar field to the exclusion object and use that as the key in the map here?